core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -213,6 +210,7 @@ public static Socket connect(Properties props, String unixPathSuffix) throws IOE
       return UnixSocketChannel.open(socketAddress).socket();
     }
 
+    final List<String> ipTypes = listIpTypes(props.getProperty("ipTypes", DEFAULT_IP_TYPES));
```

Comment:
Not important for this PR, but generally you don't need to mark local variables `final`. It's considered more noise than the extra information is worth.